afk-daily.sh

```diff
@@ -1684,11 +1685,13 @@ collectQuestChests() {
     collectQuestChests_quick
     sleep 4
 
-    inputTapSleep 650 1650 # Weeklies
+    inputTapSleep 650 1650 1 # Weeklies
+    inputTapSleep 650 1650   # Weeklies
```

Comment:
Why are we doing this for a second time?

Same goes for line `1694`.

Reply:
My inventory is full of crap, it's closing the popup telling me that it's full 😉